shared/src/androidMain/kotlin/com/kcc/kmmhackathon/shared/utility/DateKMM.kt

```diff
@@ -0,0 +1,5 @@
+package com.kcc.kmmhackathon.shared.utility
+
+import java.util.*
+
+actual typealias DateKMM = Date
```

Comment:
This java Date class fully matches the `expect` so we don't need to add anything else 🎉

Reply:
Nice!